docs/source/conf.py

```diff
@@ -0,0 +1,99 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# For the full list of built-in configuration values, see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+import os
+import sys
+import datetime
+
+sys.path.insert(0, os.path.abspath("../../"))
```

Comment:
```suggestion
sys.path.insert(0, os.path.abspath("../../src"))
```
https://cleanvision.readthedocs.io/en/uly-docs/cleanvision/utils/utils.html
Right now, `src` directory also looks like a package. This should correct that.